jina/flow/__init__.py

```diff
@@ -104,7 +104,7 @@ def index_lines(self, lines: Iterator[str] = None, filepath: str = None, size: i
         """
         from ..clients.sugary_io import _input_lines
         return self._get_client(**kwargs).index(_input_lines(lines, filepath, size, sampling_rate, read_mode),
-                                         on_done, on_error, on_always, data_type=DataInputType.CONTENT, **kwargs)
+                                         on_done, on_error, on_always, data_type=DataInputType.AUTO, **kwargs)
```

Comment:
what does AUTO do?

Reply:
It first tries `DOCUMENT` and in case it fails falls back to `CONTENT`.